quicfire_tools/inputs.py

```diff
@@ -722,3 +745,356 @@ def from_file(cls, directory: str | Path):
             maximum_firebrand_ignitions=maximum_firebrand_ignitions,
             minimum_landing_angle=minimum_landing_angle,
             maximum_firebrand_thickness=maximum_firebrand_thickness)
+
+
+class QUIC_fire(InputFile):
+    """
+    Class representing the QUIC_fire.inp input file. This file
+    contains the parameters relating to the fire simulation and
+    outputs.
+
+    Parameters
+    ----------
+    nz : int
+        Number of fire grid cells in the z-direction.
+    output_times: int | OutputTimes
+        @anthony
+    time_now : int
+        When the fire is ignited in Unix Epoch time (integer seconds since
+        1970/1/1 00:00:00). Must be greater or equal to the time of the first
+        wind
+    sim_time : int
+        Total simulation time for the fire [s]
+    fire_flag : int
+        Fire flag, 1 = run fire; 0 = no fire
+    random_seed : int
+        Random number generator, -1: use time and date, any other integer > 0
+        is used as the seed
+    fire_time_step : int
+        time step for the fire simulation [s]
+    quic_time_step : int
+        Number of fire time steps done before updating the quic wind field
+        (integer, >= 1)
+    stretch_grid_flag : int
+        Vertical stretching flag: 0 = uniform dz, 1 = custom
+    file_path : str
+        Path to files defining fuels, ignitions, and topography, with file
+        separator at the end. Defaults to "", indicating files are in the same directory as all other input files
+    dz : int
+        Cell size in the z-direction [m] of the fire grid. Recommended value: 1m
+    dz_array : list[float]
+        custom dz, one dz per line must be specified, from the ground to the
+        top of the domain
+    fuel_flag : int
```

Comment:
Nice, I like doing this as the same flag for all.

Reply:
Sounds good